docassemble/ALWeaver/validate_template_files.py

```diff
@@ -0,0 +1,171 @@
+from jinja2 import DebugUndefined
+from jinja2.utils import missing
+from docxtpl import DocxTemplate
+from docx2python import docx2python
+from jinja2 import Environment, BaseLoader
+import jinja2.exceptions
+from docassemble.base.util import DAFile
+from docassemble.base.parse import (
+    DAEnvironment,
+    DAExtension,
+    registered_jinja_filters,
+    builtin_jinja_filters,
+)
+import docassemble.base.util
+import keyword
+import docassemble.AssemblyLine.al_general
+import docassemble.AssemblyLine.al_document
+import docassemble.AssemblyLine.language
+import docassemble.AssemblyLine.sessions
```

Comment:
This line is the cause of the failure, it's importing the `init_sqlsqlachemy` which eventually tries to load the config (everything that does database stuff does). I can't find a way around it, so maybe just commenting it out for now will work?
```suggestion
# import docassemble.AssemblyLine.sessions
```

Reply:
Yeah that seems fine, these are really unlikely to lead to collisions. Good catch, thank you!